include/lbann/data_readers/sample_list_jag_impl.hpp

```diff
@@ -147,25 +147,25 @@ inline sample_list_header sample_list_jag::read_header(std::istream& istrm, cons
 }
 
 inline hid_t sample_list_jag::get_conduit_bundle_samples(std::string conduit_file_path, std::vector<std::string>& sample_names, size_t included_samples, size_t excluded_samples) {
-  hid_t hdf5_file_hnd = 0;
+  io_t file_hnd = 0;
   bool retry = false;
   int retry_cnt = 0;
   do {
     try {
-      hdf5_file_hnd = conduit::relay::io::hdf5_open_file_for_read( conduit_file_path );
+      file_hnd.open( conduit_file_path );
```

Comment:
Where does the code specify that it is manipulating an HDF5 file?

Reply:
This PR isn't actually ready for review. I'm unsure what you're asking. If you're referring to the added line, it's incomplete; should read file_hnd.open( conduit_file_path, "hdf5");